lib/ae_mdw_web/router.ex

```diff
@@ -10,7 +10,11 @@ defmodule AeMdwWeb.Router do
     {"/contracts/logs", AeMdwWeb.ContractController, :logs},
     {"/contracts/calls", AeMdwWeb.ContractController, :calls},
     {"/totalstats/", AeMdwWeb.StatsController, :total_stats},
-    {"/status", AeMdwWeb.UtilController, :status}
+    {"/status", AeMdwWeb.UtilController, :status},
+    {"/aex141", AeMdwWeb.AexnTokenController, :aex141_tokens},
+    {"/aex141/:contract_id", AeMdwWeb.AexnTokenController, :aex141_token},
+    {"/aex141/:contract_id/owner/:token_id", AeMdwWeb.Aex141Controller, :nft_owner},
+    {"/aex141/owned_nfts/:account_id", AeMdwWeb.Aex141Controller, :owned_nfts}
```

Comment:
Like with `account-balances` down below, I think we should make all of our routes use `-` instead of `_` which is the friendly URL way of formatting it